api-spec/api.yaml

```diff
@@ -273,6 +273,42 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/ProblemJson'
+    patch:
+      tags:
+        - payment-methods
+      operationId: updateSession
+      summary: Update session data
+      description: API for updating session data
+      parameters:
+        - name: id
+          in: path
+          description: Payment Method ID
+          required: true
+          schema:
+            type: string
+        - name: sessionId
+          in: path
+          description: Session payment method ID related to NPG
+          required: true
+          schema:
+            type: string
+      requestBody:
+        $ref: "#/components/requestBodies/PatchSession"
+      responses:
+        '204':
+          description: Session updated
+        '404':
```

Comment:
Add here 409 response status code returned in case transactionId is already associated to the input sessionId